test/cluster/mv/test_mv_building.py

```diff
@@ -113,3 +113,26 @@ async def test_view_building_with_tablet_move(manager: ManagerClient, build_mode
 
         for view in views:
             await wait_for_view(cql, view, len(servers))
+
+# While view building is in progress, drop the view and change the schema
+# of the base table. The view table's state may become inconsistent with
+# the base table because they are detached.
```

Comment:
Would have been nice to mention in this comment that it reproduces issue `#????`.  But to claim that, you would need to verify it really does - namely that the test really does fail (sometimes - it doesn't need to fail all the time!) before your patch, and always passes (at least trying a fairly large number of times) after your patch.